src/blockchain_db/lmdb/db_lmdb.cpp

```diff
@@ -489,7 +489,6 @@ mdb_txn_safe::~mdb_txn_safe()
 {
   if (!m_check)
     return;
-  LOG_PRINT_L3("mdb_txn_safe: destructor");
```

Comment:
These trace level logs are useful if attempting to debug something in the LMDB layer, which happens from time to time.